test/types.js

```diff
@@ -330,10 +330,10 @@ defineTypeTest('string', [
 defineTypeTest('text', [
   'TINYTEXT NULL',
   'MEDIUMTEXT NULL',
-  'LONGTEXT CHARACTER SET utf8 NULL',
+  'LONGTEXT NULL',
```

Comment:
Setting the `CHARACTER SET` is handled in the `utf8mb4` test case that it is unnecessary to perform in this case, especially since with the `charset` in the connection settings, MySQL 5.1 produces a different output than 5.5+.